### ISSUE TITLE
Does sonic support HPE Altoline 6960 Switch(JL279A)

### ISSUE THREAD

**Issue by louishot** [sonic-net/sonic-buildimage#13978]
I did not see it on the list
https://github.com/sonic-net/SONiC/blob/sonic_image_md_update/supported_devices_platforms.md
`Product Name         0x21       59  HPE Altoline 6960 32QSFP28 x86 ONIE AC Front-to-Back Switch
Part Number          0x22       13  F0P8J7632000A
Serial Number        0x23       10  **********
Base MAC Address     0x24        6  **********
Manufacture Date     0x25       19  01/26/2016 12:12:38
Label Revision       0x27        4  R01A
Platform Name        0x28       27  x86_64-accton_as7712_32x-r0
ONIE Version         0x29       13  2020.08.00.02
MAC Addresses        0x2A        2  131
Manufacturer         0x2B        6  Accton
Manufacture Country  0x2C        2  TW
Vendor Name          0x2D       26  Hewlett Packard Enterprise
Diag Version         0x2E        7  0.0.2.4`

**Comment by jeffegg** [comment]
Yes I use sonic on one, the HP is a rebranded Accton AS7712-32X AND Sonic supports this. No additional changes, I remember, I needed to make it work from the Accton code

**Comment by louishot** [comment]
> Yes I use sonic on one, the HP is a rebranded Accton AS7712-32X AND Sonic supports this. No additional changes, I remember, I needed to make it work from the Accton code
@jeffegg 
I see different things, etc memory and storage
AS7712-32X
```
Switch Silicon: Broadcom BCM56960 Tomahawk 3.2 Tbps
 No PHYs, No Retimers
 CPU Modules:
 Intel® Atom® C2538 quad-core 2.4 GHz x86 processor
 DDR3: SO-DIMM 8 GB x 2
 SPI Flash: 8 MB x 2
 mSATA SSD: 64 GB MLC
```

Altoline 6960
```
Intel Rangeley C2538 4-core @ 2.4 GHz, 8 GB DDR3 SDRAM;
Storage: mSATA: 32 GB; Packet buffer size: 12 MB, 8 GB NAND flash
```

**Comment by louishot** [comment]
@jeffegg 
Not sure why SONIC installed on `sdb` instead of `sda`
```
admin@sonic:~$ sudo fdisk -l
Disk /dev/sda: 29.5 GiB, 31675383808 bytes, 61865984 sectors
Disk model: ATP mSATA       
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: dos
Disk identifier: 0x206d72a0

Device     Boot Start      End  Sectors  Size Id Type
/dev/sda1        2048 61865983 61863936 29.5G 83 Linux


Disk /dev/sdb: 7.5 GiB, 8048869376 bytes, 15720448 sectors
Disk model: ATP IG eUSB     
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: gpt
Disk identifier: 98CEB83D-6A9B-4577-ACCB-4EEC3AD27084

Device      Start      End  Sectors  Size Type
/dev/sdb1    2048     6143     4096    2M BIOS boot
/dev/sdb2    6144   268287   262144  128M ONIE boot
/dev/sdb3  268288   530431   262144  128M Linux filesystem
/dev/sdb4  530432 15720414 15189983  7.2G Linux filesystem
```

**Comment by jeffegg** [comment]
Hmm that is different:
It looks like you are booting from a USB, do you have something in the USB port? I took mine apart when I got it, I cannot remember if there was an internal usb connector or not.
Disk /dev/sdb: 7.5 GiB, 8048869376 bytes, 15720448 sectors
Disk model: **ATP IG eUSB**     

Here is my dump, we just see /dev/sda in mine
admin@sonic:~$ sudo fdisk -l
Disk /dev/sda: 29.82 GiB, 32017047552 bytes, 62533296 sectors
Disk model: TS32EPTMM1600
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes
Disklabel type: gpt
Disk identifier: 6FAA0FF2-9DF0-46A1-AE0A-1F28CFE6C9D3

Device      Start      End  Sectors  Size Type
/dev/sda1    2048     6143     4096    2M BIOS boot
/dev/sda2    6144   268287   262144  128M ONIE boot
/dev/sda3  268288   530431   262144  128M Linux filesystem
/dev/sda4  530432 61499966 60969535 29.1G Linux filesystem


Disk /dev/loop0: 348.36 MiB, 365277184 bytes, 713432 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes


Disk /dev/loop1: 4 GiB, 4294967296 bytes, 8388608 sectors
Units: sectors of 1 * 512 = 512 bytes
Sector size (logical/physical): 512 bytes / 512 bytes
I/O size (minimum/optimal): 512 bytes / 512 bytes

**Comment by louishot** [comment]
@jeffegg 
Yes this one was an internal usb, looks like an eUSB mean embedded USB module
I'll try reinstall ONIE on sda
```
Disk /dev/sdb: 7.5 GiB, 8048869376 bytes, 15720448 sectors
Disk model: ATP IG eUSB
```

**Comment by judyjoseph** [comment]
@jostar-yang Could you further check for this platform and close this issue if the platform is supported.
